package classwork.Exceptions;

public class Exception6Example {
//    public static void main(String[] args) {
//        try{
//            throw new RuntimeException();
//        } catch (RuntimeException e){
//            System.out.println("Catch block ");
//        }
//        System.out.println("Between catch and finally");
//        finally{
//            System.out.println("Finally block ");
//        }
//    }
}
